index.html

```diff
@@ -1,83 +1,94 @@
 <!doctype html>
 <html ng-app="app1">
-	<head>
-		<link rel="stylesheet" href="styles/namelist.css">
-		<link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
-		<link rel="stylesheet" href="styles/general.css">
-		<title>Student reports generator</title>
-	</head>
-	<body>
-		<div class="container">
-			<div class="page-header">
-				<h1>Student reports generator</h1>
-			</div>
+<head>
```

Comment:
Speaking of editors, do you think it is a potential problem all 3 of us are using different editors? I suppose if we are all using the config that @WillMoreland set up, probably not.